scripts/biomedical/Human-GEM_Human1/scripts/format_groups.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
Google license and header missing from all script files

Reply:
added